modules/claims_api/config/schemas/v2/poa_request.json

```diff
@@ -0,0 +1,264 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "description": "POA Request (21-22/a) Schema",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "veteran",
+    "poa",
+    "recordConsent",
+    "consentAddressChange"
+  ],
+  "properties": {
+    "veteran": {
+      "type": "object",
+      "additionalProperties": false,
+      "required": [
+        "address"
+      ],
+      "properties": {
+        "serviceNumber": {
+          "description": "The Veteran's Service Number",
+          "type": "string",
+          "maxLength": 9
+        },
+        "serviceBranch": {
```

Comment:
the `serviceBranchOther` attribute has been removed for now since we do not know which values are supported by BGS at this time